SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -789,6 +789,647 @@ void CFlowOutput::Set_CpInverseDesign(CSolver *solver, const CGeometry *geometry
+
+            su2double FixAzimuthalLine = config->GetFixAzimuthalLine();
+
+            if ((AngleDouble >= FixAzimuthalLine - 0.1) && (AngleDouble <= FixAzimuthalLine + 0.1)) AngleDouble = FixAzimuthalLine - 0.1;
+
+            AngleInt = SU2_TYPE::Short(floor(AngleDouble + 0.5));
+            if (AngleInt >= 0) AzimuthalAngle[nVertex_NearField] = AngleInt;
+            else AzimuthalAngle[nVertex_NearField] = 180 + AngleInt;
+          }
+
+          if (AzimuthalAngle[nVertex_NearField] <= 60) {
+            Pressure[nVertex_NearField] = solver->GetNodes()->GetPressure(iPoint);
+            FaceArea[nVertex_NearField] = fabs(Face_Normal[nDim-1]);
+            nVertex_NearField ++;
+          }
+
+        }
+      }
+
+#else
+
+  int nProcessor;
+  SU2_MPI::Comm_size(SU2_MPI::GetComm(), &nProcessor);
+
+  unsigned long nLocalVertex_NearField = 0, MaxLocalVertex_NearField = 0;
+  int iProcessor;
+
+  unsigned long *Buffer_Receive_nVertex = NULL;
+  if (rank == MASTER_NODE) {
+    Buffer_Receive_nVertex = new unsigned long [nProcessor];
+  }
+
+  /*--- Compute the total number of points of the near-field ghost nodes ---*/
+
+  nLocalVertex_NearField = 0;
+  for (iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++)
+    if (config->GetMarker_All_KindBC(iMarker) == NEARFIELD_BOUNDARY)
+      for (iVertex = 0; iVertex < geometry->GetnVertex(iMarker); iVertex++) {
+        iPoint = geometry->vertex[iMarker][iVertex]->GetNode();
+        Face_Normal = geometry->vertex[iMarker][iVertex]->GetNormal();
+        Coord = geometry->nodes->GetCoord(iPoint);
+
+        if (geometry->nodes->GetDomain(iPoint))
+          if ((Face_Normal[nDim-1] > 0.0) && (Coord[nDim-1] < 0.0))
+            nLocalVertex_NearField ++;
+      }
+
+  unsigned long *Buffer_Send_nVertex = new unsigned long [1];
+  Buffer_Send_nVertex[0] = nLocalVertex_NearField;
+
+  /*--- Send Near-Field vertex information --*/
+
+  SU2_MPI::Allreduce(&nLocalVertex_NearField, &nVertex_NearField, 1, MPI_UNSIGNED_LONG, MPI_SUM, SU2_MPI::GetComm());
+  SU2_MPI::Allreduce(&nLocalVertex_NearField, &MaxLocalVertex_NearField, 1, MPI_UNSIGNED_LONG, MPI_MAX, SU2_MPI::GetComm());
+  SU2_MPI::Gather(Buffer_Send_nVertex, 1, MPI_UNSIGNED_LONG, Buffer_Receive_nVertex, 1, MPI_UNSIGNED_LONG, MASTER_NODE, SU2_MPI::GetComm());
+  delete [] Buffer_Send_nVertex;
+
+  su2double *Buffer_Send_Xcoord = new su2double[MaxLocalVertex_NearField];
+  su2double *Buffer_Send_Ycoord = new su2double[MaxLocalVertex_NearField];
+  su2double *Buffer_Send_Zcoord = new su2double[MaxLocalVertex_NearField];
+  unsigned long *Buffer_Send_IdPoint = new unsigned long [MaxLocalVertex_NearField];
+  su2double *Buffer_Send_Pressure = new su2double [MaxLocalVertex_NearField];
+  su2double *Buffer_Send_FaceArea = new su2double[MaxLocalVertex_NearField];
+
+  su2double *Buffer_Receive_Xcoord = NULL;
+  su2double *Buffer_Receive_Ycoord = NULL;
+  su2double *Buffer_Receive_Zcoord = NULL;
```

Comment:
`nullptr` instead of NULL (but std::vector would save all that)

Reply:
See below.
https://github.com/su2code/SU2/pull/1329#discussion_r674043743

Reply:
Referencing a comment 😮 ... i didn't know that was possible